_opensearch/rest-api/index-apis/dangling-index.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: default
+title: Dangling indices
+parent: index-apis
+grand_parent: REST API reference
+nav_order: 84
+---
+
+# Dangling indices API
+
+Dangling indices occur if any shards exist in a node's local directory that do not already exist in the cluster, after a node joins a cluster. Dangling indices can be listed, deleted, or imported.
```

Comment:
I think the first sentence might be clearer if the prepositional phrase comes first: "After a node joins a cluster, dangling indexes occur if any shards exist in the node's local directory that do not already exist in the cluster."